.travis.yml

```diff
@@ -18,3 +18,9 @@ deploy:
     node: '4'
     tags: true
     repo: thelounge/lounge
+
+script:
```

Comment:
Hmm, it does run it and hides the output. Do we know if travis fails the build if grunt/handlebars build fails at that stage?